exercises/react/src/lib.rs

```diff
@@ -99,6 +97,10 @@ impl <T: Copy + PartialEq> Reactor<T> {
     //
     // A removed callback should no longer be called.
     pub fn remove_callback(&mut self, cell: CellID, callback: CallbackID) -> Result<(), RemoveCallbackError> {
-        unimplemented!()
+        unimplemented!(
+            "Remove the callback identified by the CallbackId {:?} from the cell {:?}",
```

Comment:
`CallbackId`'s capitalisation is inconsistent with the fact that `CallbackID` is used a few lines above